samples/bluetooth/peripheral_uart/boards/thingy53_nrf5340_cpuapp.conf

```diff
@@ -59,22 +59,15 @@ CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
 CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=16
 
 ################################################################################
-# Enable USB CDC ACM
+# Configure USB CDC ACM
 
-CONFIG_USB_DEVICE_STACK=y
-CONFIG_USB_DEVICE_REMOTE_WAKEUP=n
 CONFIG_USB_DEVICE_PRODUCT="Thingy:53 Nordic UART Service sample"
-CONFIG_USB_DEVICE_VID=0x1915
 CONFIG_USB_DEVICE_PID=0x530E
 CONFIG_USB_COMPOSITE_DEVICE=y
-CONFIG_USB_CDC_ACM=y
-CONFIG_USB_CDC_ACM_LOG_LEVEL_ERR=y
-CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
 
-CONFIG_LOG_BACKEND_UART=y
 CONFIG_LOG_PRINTK=y
 CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
```

Comment:
Do we need to configure these explicitly? I think we should ensure that the USB CDC configuration would work out-of-the-box also for other samples (that rely on default values here).

Reply:
I would opt for removing all of the unnecessary configurations related to logging or USB CDC ACM to simplify the user experience (if something is mandatory we should probably set it by default in Thingy:53 board definition files).

Reply:
Removed all unnecessary configs